src/components/pages/exportcontent/contentcatalogue.js

```diff
@@ -0,0 +1,30 @@
+import React, { useState, useEffect } from 'react';
+import { useLocation } from '@reach/router';
+import { buildQueryString, parseQueryString } from '~utils/query-string';
+
+const SIMPLESEARCHURL = `https://us-east-1.aws.webhooks.mongodb-realm.com/api/client/v2.0/app/devhub-search-service-dxfcg/service/devhub_export/incoming_webhook/simplesearch`;
+const COMPOSITESEARCHURL = `https://us-east-1.aws.webhooks.mongodb-realm.com/api/client/v2.0/app/devhub-search-service-dxfcg/service/devhub_export/incoming_webhook/compositesearch`;
```

Comment:
What is the difference between the simple search and composite search? May be worth adding a comment to clarify

Reply:
PR description has the description for you, however I have a doubt on this: Personally I'm not a fan of switching between two calls depending on parameters passed and I prefer going with separate pages for each type of search, but I also think Sheeri might say we don't even need a simple search